Documentation/security/restrict-pod-access.rst

```diff
@@ -0,0 +1,52 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _security_restrict_pod_access:
+
+========================================
+Restricting privileged Cilium pod access
+========================================
+
+This page shows you how to restrict priveleged access to Cilium pods by limiting access from the Kubernetes API, specifically from `kubernetes exec pod`_.
+
+.. include:: gsg_requirements.rst
+
+Background
+----------
+
+While the Cilium agent no longer requires a specific set of privileges and capabilities, the Cilium agent requires specific privileges to perform essential system and network operations. 
+
+Cilium relies on Kubernetes and Containers to mount filesystems and the environment. Cilium doesn't perform any extra operations that could result in an unsafe volume mount. 
+
+Cilium needs kernel interfaces to properly configure the environment. Some kernel interfaces are part of the ``/proc`` filesystem, which includes host and machine configurations that can't be virtualized or namespaced.
+
+If ``pod exec`` operations aren't restricted, then remote `exec into pods`_ and containers defeats restrictions on Linux namespaces and capabilities.
+
+Entering a container that runs with execution (``X``) privileges like Cilium requires the ``CAP_SYS_ADMIN ``capability in your current user namespace and in the target Cilium user namespace. Because Cilium runs in the initial user namespace, it requires ``CAP_SYS_ADMIN`` capability, which provides essentially full root access.
+
+To prevent attacks through privileged access to the Cilium pod, restrict authorization access to the Kubernetes API and arbitrary ``pod exec`` operations.
```

Comment:
To prevent privileged Cilium pod access, restrict access to the Kubernetes API and arbitrary ``pod exec`` operations.

Reply:
> To prevent privileged Cilium pod access

It's not clear what "privileged" points to here. Is it accurate to say:

> To prevent privileged access to Cilium pods, restrict access to the Kubernetes API and arbitrary ``pod exec`` operations.

?

Reply:
Yes!